docs/book/src/topics/addons.md

```diff
@@ -10,29 +10,50 @@ To install [Calico](https://www.tigera.io/project-calico/) on a self-managed clu
 
 ### For IPv4 clusters
 
+On the management cluster:
```

Comment:
Rather than "On the management cluster" maybe we want to clarify that we want the `kubectl` statement to point to the management cluster. Maybe:

"Grab the IPv4 CIDR from your newly created cluster by running this kubectl statement against the Cluster API management cluster:"